businessRules/donations.js

```diff
@@ -55,10 +56,32 @@ function createDonationByRegisteredUser(ideaId, userId, sum) {
 function getTotalSumDonatedForIdea(ideaId) {
     const idea = getIdea(ideaId);
 
-    return idea.totalDonationSum;
+    return {
+        numberOfDonations: idea.totalDonationSum,
+    };
```

Comment:
The function is called `getTotalSumDonatedForIdea`, but you're returning `numberOfDonations`?

Reply:
Fixed.